server/requirements.txt

```diff
@@ -1,4 +1,4 @@
-anndata>=0.6.20
+anndata>=0.7
```

Comment:
Not sure if this is legit or not, but does this need to be the full semvar in order for the comparison to work? i.e. 0.7.0?

Reply:
it worked locally but I can update it just to be safe